scipy/sparse/linalg/isolve.py

```diff
@@ -2,14 +2,25 @@
 # Use the `scipy.sparse.linalg` namespace for importing the functions
 # included below.
 
-import warnings
-from . import _isolve
+from scipy._lib.deprecation import _sub_module_deprecation
 
 
 __all__ = [  # noqa: F822
-    'bicg', 'bicgstab', 'cg', 'cgs', 'gcrotmk', 'gmres',
-    'lgmres', 'lsmr', 'lsqr',
-    'minres', 'qmr', 'tfqmr', 'utils', 'iterative', 'test'
+    'bicg', 
+    'bicgstab', 
+    'cg', 
+    'cgs', 
+    'gcrotmk', 
+    'gmres',
+    'lgmres', 
+    'lsmr',
+    'lsqr',
+    'minres',
+    'qmr',
+    'tfqmr', 
+    'utils', 
+    'iterative', 
+    'test'
```

Comment:
and here

Reply:
again please revert this change it makes it harder to tell if anything has been removed